### PR TITLE
Add Aware.app v1.0.3

### PR DESCRIPTION
**Pull request by ttaylorr**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

----

/cc @josh

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

**Comment by miccal** [comment]
You need a [comment](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment) above the `url` stanza.

Use the `version` in their like so:
```
url 'https://github.com/josh/Aware/releases/download/v#{version}/Aware.zip'
```

Finally, could you please add an [appcast](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md)?

**Comment by josh** [comment]
Cool, but what's wrong with downloading from the Mac App Store? Its free.

File: Casks/aware.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'aware' do
+  version '1.0.3'
+  sha256 '65dcaef6735c649e2b6a406f48583609ca9d66e7680ca69eb61871502613dec3'
+
+  url 'https://github.com/josh/Aware/releases/download/v1.0.3/Aware.zip'
```
Comment (vitorgalvao) [inline Casks/aware.rb]: This needs the comment, like @miccal. It’s unlikely this has passed `style`.

Comment (vitorgalvao) [inline Casks/aware.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

Comment (vitorgalvao) [inline Casks/aware.rb]: `url "https://github.com/josh/Aware/releases/download/v#{version}/Aware.zip"`

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
@josh Many people avoid the MAS like the plague and prefer to download it directly. Also, if you already manage all your apps via Homebrew-Cask, you want as many of them to be installable in this manner.

File: Casks/aware.rb (hunk 1 above)
Comment (ttaylorr) [inline Casks/aware.rb]: Ah, I changed the URL and squashed it into the initial commit I made, without re-running `style`. It did indeed catch it, and is fixed as of https://github.com/caskroom/homebrew-cask/pull/26296/commits/95dbde348e0c9cdc3337c583d7bd2a99f99f7df9.

Comment (ttaylorr) [inline Casks/aware.rb]: https://github.com/caskroom/homebrew-cask/pull/26296/commits/e5ac0df1435cfa374c0c6f7942f3d859ca805de7.

File: Casks/aware.rb [hunk 2]
```diff
@@ -2,7 +2,10 @@
   version '1.0.3'
   sha256 '65dcaef6735c649e2b6a406f48583609ca9d66e7680ca69eb61871502613dec3'
 
+  # github.com was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/aware.rb]: Needs to be more specific: `github.com/josh/Aware`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/aware.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+cask 'aware' do
+  version '1.0.3'
+  sha256 '65dcaef6735c649e2b6a406f48583609ca9d66e7680ca69eb61871502613dec3'
+
+  # github.com was verified as official when first introduced to the cask
+  url 'https://github.com/josh/Aware/releases/download/v1.0.3/Aware.zip'
```
Comment (vitorgalvao) [inline Casks/aware.rb]: Please make the change requested by @miccal.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/aware.rb (hunk 1 above)
Comment (ttaylorr) [inline Casks/aware.rb]: Nice. 1482ba602742652dffd263fad28b58c0f05fac09.

File: Casks/aware.rb (hunk 2 above)
Comment (ttaylorr) [inline Casks/aware.rb]: Was referencing Atom's [cask](https://github.com/caskroom/homebrew-cask/blob/161f85b605e160ff96e7dd11732d85609e13dc51/Casks/atom.rb#L5). See: bbc3b99c88a43d699848047f0fdb90993b2eff53.

**Review by vitorgalvao — APPROVED** [review]

**Comment by vitorgalvao** [comment]
Thank you for the contribution and fast changes.

**Comment by ttaylorr** [comment]
The test failure appears unrelated.

File: Casks/aware.rb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+cask 'aware' do
+  version '1.0.3'
+  sha256 '65dcaef6735c649e2b6a406f48583609ca9d66e7680ca69eb61871502613dec3'
+
+  # github.com/josh/Aware was verified as official when first introduced to the cask
+  url "https://github.com/josh/Aware/releases/download/v#{version}/Aware.zip"
+  appcast 'https://github.com/jos/Aware/releases.atom',
```
Comment (vitorgalvao) [inline Casks/aware.rb]: This is wrong (`jos` instead of `josh`). I’ll fix it.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 187cc3046eb7acbf5363218692519d98e607d199. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**